src/renderer/screens/manager/FirmwareUpdate/index.js

```diff
@@ -20,6 +20,8 @@ import type { ModalStatus } from "./types";
 type Props = {
   deviceInfo: DeviceInfo,
   device: Device,
+  onUpdateStart: () => void,
```

Comment:
are they always required ? Are you sure there are no other context in which onUpdateStart and onUpdateStop will not get passed ?

Reply:
My bad I thought it was the Modal